Content.Client/Weapons/Melee/UI/MeleeSpeechWindow.xaml.cs

```diff
@@ -0,0 +1,28 @@
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Client.UserInterface.XAML;
+
+namespace Content.Client.Weapons.Melee.UI
```

Comment:
```suggestion
namespace Content.Client.Weapons.Melee.UI;
```

Same as above, convert to declared namespace and remove extra {}